### ISSUE TITLE
Unable to make it run

### ISSUE THREAD

**Issue by isaacgg** [maartenbreddels/ipyvolume#203]
Hi, I installed this library weeks ago and make it work, but I accidentally deleted my enviroment and I can't make it work now. I have tried default pip version, default conda version pip 0.4.0 etc..

What happens is that the code doesn't complain, no error messages or warnings but jupyter notebook doesn't show anything at all. 
I think I may be missing some configuration step or something, not sure... because I'm using the same code I that used to work in my old enviroment.

I'm working with jupyter in anaconda and windows10.

Thanks and great work!

**Comment by maartenbreddels** [comment]
I assume classical notebook, not jupyter lab right?
Can you give me the output or a screenshot (color helps) of:
```
$ jupyter nbextension list --log-level=DEBUG
```

**Comment by maartenbreddels** [comment]
@BastiaanBergman this config looks good, do you happen to use a different kernel than the environment that the jupyter notebook is running in? (For instance, jupyter notebook in the root env, while ipyvolume only installed in the env of the kernel?).

**Comment by maartenbreddels** [comment]
> Thanks man, it looks really awesome! Can't wait to use it.

Thanks :)

**Comment by BastiaanBergman** [comment]
It was running in the right kernel, and I did restart the kernel after installation. Restarting the notebook server did it, so it's working now!
Thanks!

**Comment by maartenbreddels** [comment]
It usually does not need a server restart, a page refresh should do it. Happy to hear it is working though!.
Hope @isaacgg got it working as well.

**Comment by isaacgg** [comment]
> I assume classical notebook, not jupyter lab right?
> Can you give me the output or a screenshot (color helps) of:
> 
> ```
> $ jupyter nbextension list --log-level=DEBUG
> ```

Hey, sorry for the late reply. This is my output

`Searching ['D:\\TrabajoFinDeMaster', 'C:\\Users\\isaac\\.jupyter', 'C:\\Users\\isaac\\Anaconda3\\envs\\tensorflow-gpu-safe\\etc\\jupyter', 'C:\\ProgramData\\jupyter'] for config files
Looking for jupyter_config in C:\ProgramData\jupyter
Looking for jupyter_config in C:\Users\isaac\Anaconda3\envs\tensorflow-gpu-safe\etc\jupyter
Looking for jupyter_config in C:\Users\isaac\.jupyter
Looking for jupyter_config in D:\TrabajoFinDeMaster
Looking for jupyter nbextension list_config in C:\ProgramData\jupyter
Looking for jupyter nbextension list_config in C:\Users\isaac\Anaconda3\envs\tensorflow-gpu-safe\etc\jupyter
Looking for jupyter nbextension list_config in C:\Users\isaac\.jupyter
Looking for jupyter nbextension list_config in D:\TrabajoFinDeMaster
Known nbextensions:
Paths used for configuration of common:
        C:\Users\isaac\.jupyter\nbconfig\common.json
Paths used for configuration of notebook:
        C:\Users\isaac\.jupyter\nbconfig\notebook.json
  config dir: C:\Users\isaac\.jupyter\nbconfig
    notebook section
      ipyvolume/extension enabled
      - Validating: problems found:
        - require?  X ipyvolume/extension
      jupyter-js-widgets/extension enabled
      - Validating: ok
Paths used for configuration of tree:
        C:\Users\isaac\.jupyter\nbconfig\tree.json
Paths used for configuration of edit:
        C:\Users\isaac\.jupyter\nbconfig\edit.json
Paths used for configuration of terminal:
        C:\Users\isaac\.jupyter\nbconfig\terminal.json
Paths used for configuration of common:
        C:\Users\isaac\Anaconda3\envs\tensorflow-gpu-safe\etc\jupyter\nbconfig\common.json
Paths used for configuration of notebook:
        C:\Users\isaac\Anaconda3\envs\tensorflow-gpu-safe\etc\jupyter\nbconfig\notebook.d\ipywebrtc.json
        C:\Users\isaac\Anaconda3\envs\tensorflow-gpu-safe\etc\jupyter\nbconfig\notebook.d\jupyter-datawidgets.json
        C:\Users\isaac\Anaconda3\envs\tensorflow-gpu-safe\etc\jupyter\nbconfig\notebook.d\jupyter-threejs.json
        C:\Users\isaac\Anaconda3\envs\tensorflow-gpu-safe\etc\jupyter\nbconfig\notebook.d\widgetsnbextension.json
        C:\Users\isaac\Anaconda3\envs\tensorflow-gpu-safe\etc\jupyter\nbconfig\notebook.json
  config dir: C:\Users\isaac\Anaconda3\envs\tensorflow-gpu-safe\etc\jupyter\nbconfig
    notebook section
      jupyter-threejs/extension enabled
      - Validating: ok
      jupyter-webrtc/extension enabled
      - Validating: ok
      jupyter-datawidgets/extension enabled
      - Validating: ok
      jupyter-js-widgets/extension enabled
      - Validating: ok
Paths used for configuration of tree:
        C:\Users\isaac\Anaconda3\envs\tensorflow-gpu-safe\etc\jupyter\nbconfig\tree.json
Paths used for configuration of edit:
        C:\Users\isaac\Anaconda3\envs\tensorflow-gpu-safe\etc\jupyter\nbconfig\edit.json
Paths used for configuration of terminal:
        C:\Users\isaac\Anaconda3\envs\tensorflow-gpu-safe\etc\jupyter\nbconfig\terminal.json
Paths used for configuration of common:
        C:\ProgramData\jupyter\nbconfig\common.json
Paths used for configuration of notebook:
        C:\ProgramData\jupyter\nbconfig\notebook.json
Paths used for configuration of tree:
        C:\ProgramData\jupyter\nbconfig\tree.json
Paths used for configuration of edit:
        C:\ProgramData\jupyter\nbconfig\edit.json
Paths used for configuration of terminal:
        C:\ProgramData\jupyter\nbconfig\terminal.json`

**Comment by maartenbreddels** [comment]
There is something funny going on, how you installed ipyvolume.
I see you are using the anaconda env tensorflow-gpu-safe, and in that, I see ipywebrtc are installed and enabled (C:\Users\isaac\Anaconda3\envs\tensorflow-gpu-safe\etc\jupyter\nbconfig\notebook.d\ipywebrtc.json)
But ipyvolume is enabled as user (In config dir: C:\Users\isaac\.jupyter\nbconfig). 
How did you install ipyvolume (pip install --user maybe?)? I think you should reinstall it and you should see the same .json file for ipyvolume is this log.

**Comment by isaacgg** [comment]
Yes! I used --user because I was getting a permission denied error without it. I'm going to try again
Thanks!

**Comment by isaacgg** [comment]
Ok, I was able to install with pip it but now I'm having this `ImportError: cannot import name 'Video'` when I do `import ipyvolume`
I have installed ipyvolume 0.5.1

**Comment by maartenbreddels** [comment]
Upgrade ipywidgets, that should help I think

(from mobile phone)

**Comment by isaacgg** [comment]
It turns out that my anaconda installation was pretty messed up. Fresh installation solved the problem. Thanks!

**Comment by maartenbreddels** [comment]
Great to hear it is working, enjoy ipyvolume!

(from mobile phone)

**Comment by sk-drop** [comment]
Hey @maartenbreddels - we're trying to:

`!jupyter nbextension enable --debug --py --user ipyvolume`
but we're getting:
`Searching ['/Users/syrus/Desktop/ELK/geoviz', '/Users/syrus/.jupyter', '/usr/local/Cellar/python@3.9/3.9.2_1/Frameworks/Python.framework/Versions/3.9/etc/jupyter', '/usr/local/etc/jupyter', '/etc/jupyter'] for config files
Looking for jupyter_config in /etc/jupyter
Looking for jupyter_config in /usr/local/etc/jupyter
Looking for jupyter_config in /usr/local/Cellar/python@3.9/3.9.2_1/Frameworks/Python.framework/Versions/3.9/etc/jupyter
Looking for jupyter_config in /Users/syrus/.jupyter
Looking for jupyter_config in /Users/syrus/Desktop/ELK/geoviz
Looking for jupyter_notebook_config in /etc/jupyter
Looking for jupyter_notebook_config in /usr/local/etc/jupyter
Looking for jupyter_notebook_config in /usr/local/Cellar/python@3.9/3.9.2_1/Frameworks/Python.framework/Versions/3.9/etc/jupyter
Looking for jupyter_notebook_config in /Users/syrus/.jupyter
Looking for jupyter_notebook_config in /Users/syrus/Desktop/ELK/geoviz
Enabling notebook extension ipyvolume/extension...
Paths used for configuration of notebook: 
    	/Users/syrus/.jupyter/nbconfig/notebook.json
Paths used for configuration of notebook: 
    	
      - Validating: problems found:
        - require?  X ipyvolume/extension
Paths used for configuration of notebook: 
    	/Users/syrus/.jupyter/nbconfig/notebook.json`

-- we tried literally every possible solution from any forum we could find. Thanks for the help.

**Comment by maartenbreddels** [comment]
I don't think you should use `--user`, that might be broken (not 100% sure). But use pipenv/venv/conda instead and you shouldn't have issues. Hope that helps!
